bson/binary.py

```diff
@@ -549,6 +581,33 @@ def subtype(self) -> int:
         """Subtype of this binary data."""
         return self.__subtype
 
+    def as_numpy_vector(self) -> BinaryVector:
```

Comment:
Should this be a new method or a new argument to the existing as_vector method? Like `binary.as_vector(numpy=True)`